diagnostics/inspect.py

```diff
@@ -0,0 +1,91 @@
+# Copyright 2020 Google LLC
```

Comment:
Let's not pick module names overlapping with the stdlib.

Reply:
Done.